devops/scripts/benchmarks/benches/compute.py

```diff
@@ -301,6 +306,27 @@ def __init__(self, bench, name, test, runtime: RUNTIMES = None):
         self.bench_name = name
         self.test = test
         self.runtime = runtime
+        # Mandatory per-benchmark iteration counts.
+        # Subclasses MUST set both `self.iterations_regular` and
+        # `self.iterations_trace` (positive ints) in their __init__ before
+        # calling super().__init__(). The base class enforces this.
+        if (
+            not hasattr(self, "iterations_regular")
+            or not isinstance(getattr(self, "iterations_regular", None), int)
+            or getattr(self, "iterations_regular", 0) <= 0
+        ):
+            raise ValueError(
+                f"{self.bench_name}: subclasses must set `iterations_regular` (positive int) before calling super().__init__"
+            )
```

Comment:
This logic is repeated. Please, make it into some validation method, like self._validate_iterations(attr_name)